packages/bundle-source/package.json

```diff
@@ -10,9 +10,6 @@
     "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
     "test:xs": "exit 0",
     "lint-fix": "eslint --fix '**/*.js'",
-    "lint-check": "yarn lint",
-    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.js'",
-    "lint-check-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.js'",
```

Comment:
Where did these go?

Reply:
I saw no references so assumed they weren't used. Incorrect?

Reply:
Ping @michaelfig I assume?

Reply:
yes, `plugin:@jessie.js/recommended` along with `@jessie-check` is the current mechanism.